src/libmongoc/doc/errors.rst

```diff
@@ -31,6 +31,8 @@ Many C Driver functions report errors by returning ``false`` or -1 and filling o
 +-----------------------------------------+---------------------------------------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 |                                         | ``MONGOC_ERROR_CLIENT_INVALID_ENCRYPTION_STATE``                                                                                | Failure related to Client-Side Field Level Encryption.                                                                                                                                                                                                                                                                                     |
 +-----------------------------------------+---------------------------------------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|                                         | ``MONGOC_ERROR_CLIENT_INVALID_LOAD_BALANCER``                                                                                   | You attempted to connect to a MongoDB server behind a load balancer, but the server does not advertise load balanced support.                                                                                                                                                                                                              |
```

Comment:
British spelling for `advertise` but I assume this error message was copied from somewhere.

Reply:
TIL I've been using the British spelling. Updated.